config/types/files.go

```diff
@@ -187,6 +214,9 @@ func init() {
 			out.Storage.Files = append(out.Storage.Files, newFile)
 		}
 		for _, dir := range in.Storage.Directories {
+			if dir.Mode == nil {
+				dir.Mode = util.IntToPtr(0755)
```

Comment:
How about making these named constants, and then using the constants when building the `report.Entry` message?

Reply:
done

Reply:
I think he meant making the default modes constants and building the error message from them.

Reply:
Updated. Does that look like overkill?

Reply:
You should use the constant here as well

Reply:
whoops, yeah. Done.